### PR TITLE
Add syntactic sugar for bitwise operations

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: okio/jvm/src/main/java/okio/Buffer.kt [hunk 1]
```diff
@@ -1034,7 +1034,7 @@ class Buffer : BufferedSource, BufferedSink, Cloneable, ByteChannel {
     val data = tail.data
     var limit = tail.limit
     data[limit++] = (s.ushr(8) and 0xff).toByte()
-    data[limit++] = (s and 0xff).toByte()
+    data[limit++] = (s                  and 0xff).toByte()
```
Comment (Egorand) [inline okio/jvm/src/main/java/okio/Buffer.kt]: This looks weird

File: okio/jvm/src/main/java/okio/Buffer.kt [hunk 2]
```diff
@@ -1061,19 +1061,20 @@ class Buffer : BufferedSource, BufferedSink, Cloneable, ByteChannel {
...
+    data[limit++] = (v ushr 24 and 0xffL).toByte()
+    data[limit++] = (v ushr 16 and 0xffL).toByte()
+    data[limit++] = (v ushr  8 and 0xffL).toByte()
+    data[limit++] = (v         and 0xffL).toByte()
     tail.limit = limit
     size += 8L
     return this
   }
 
+
```
Comment (Egorand) [inline okio/jvm/src/main/java/okio/Buffer.kt]: nit: empty line

File: okio/jvm/src/main/java/okio/Okio.kt [hunk 3]
```diff
@@ -213,3 +213,23 @@ fun Path.source(vararg options: OpenOption): Source =
 internal val AssertionError.isAndroidGetsocknameError: Boolean get() {
   return cause != null && message?.contains("getsockname failed") ?: false
 }
+
+// TODO(jwilson): move these to Util.kt once that’s Kotlin?
```
Comment (Egorand) [inline okio/jvm/src/main/java/okio/Okio.kt]: 👍

File: okio/jvm/src/main/java/okio/Buffer.kt (hunk 1 above)
Comment (swankjesse) [reply okio/jvm/src/main/java/okio/Buffer.kt]: Umm it was an honest mistake?
screen shot 2018-04-30 at 8 50 41 am

Comment (swankjesse) [reply okio/jvm/src/main/java/okio/Buffer.kt]: Fixed. `ushr` wanted to be infix!

File: okio/jvm/src/main/java/okio/Buffer.kt (hunk 2 above)
Comment (swankjesse) [reply okio/jvm/src/main/java/okio/Buffer.kt]: Fixed.

File: okio/jvm/src/main/java/okio/Buffer.kt (hunk 1 above)
Comment (Egorand) [reply okio/jvm/src/main/java/okio/Buffer.kt]: Ah, I see. I don't like parameter name hints precisely because they screw up formatting.

**Pull request merged**